tools/scripts/build_ubuntu_x64_ncnn.py

```diff
@@ -175,7 +176,7 @@ def main():
     if install_mmdeploy(work_dir, dep_dir, ncnn_cmake_dir) != 0:
         return -1
 
-    if os.path.exists('~/mmdeploy.env'):
+    if os.path.exists(Path('~/mmdeploy.env').expanduser()):
```

Comment:
you can use `os.path.expanduser()` instead of `pathlib`

Reply:
https://pythondict.com/python-basic-tutorial/pathlib-vs-os-python/   They are same thing.